### PR TITLE
Update "add a passkey document" with more detail and addressing highlighted text

### PR DESCRIPTION
**Pull request by turnipdabeets**
Add information about passkey creation for both HWA and Embedded SDK

Direct link: https://developer-docs-git-add-a-passkey-update-beyondidentity.vercel.app/docs/next/add-passkey

### CODE REVIEW

**Comment by turnipdabeets** [comment]
> I like the "Coming soon!" copy over the "To do" copy, but I wonder if we should hide the Federated IdP and FIDO2 pages from the sidebars for now.

I could go either way. It's nice to see they are coming soon in a way even if they don't have content yet.

**Comment by nikhilkharebyid** [comment]
> > I like the "Coming soon!" copy over the "To do" copy, but I wonder if we should hide the Federated IdP and FIDO2 pages from the sidebars for now.
> 
> I could go either way. It's nice to see they are coming soon in a way even if they don't have content yet.

Yea Coming soon is better. I'm fine with hiding both Federated IDP and FIDO2 pages for now. We still need to work out the naming.

**Review by Will5 — APPROVED** [review]

File: docs/authentication/embedded-sdk-add-otp-email.mdx [hunk 1]
```diff
@@ -22,7 +22,7 @@ import { Alert, Col, Row } from "antd";
 
 <Row>
   <Col span={6}>
-    <Alert message="To do" type="warning" />
+    <Alert message="Coming soon!" type="warning" />
```
Comment (Will5) [inline docs/authentication/embedded-sdk-add-otp-email.mdx:25]: I think you missed `docs/authentication/hosted-add-otp-email.mdx`

File: docs/authentication/hosted-add-fido2.mdx [hunk 2]
```diff
@@ -22,7 +22,7 @@ import { Alert, Col, Row } from "antd";
 
 <Row>
   <Col span={6}>
-    <Alert message="To do" type="warning" />
+    <Alert message="Coming soon!" type="warning" />
```
Comment (Will5) [inline docs/authentication/hosted-add-fido2.mdx:25]: I think you missed `docs/authentication/embedded-sdk-add-fido2.mdx`

**Pull request merged**